packages/backend-core/src/constants/db.ts

```diff
@@ -1,5 +1,5 @@
-export const SEPARATOR = "_"
-export const UNICODE_MAX = "\ufff0"
+import { prefixed, DocumentType } from "@budibase/types"
+export { SEPARATOR, UNICODE_MAX, DocumentType } from "@budibase/types"
```

Comment:
Can we namespace or rename the `SEPARATOR` in here to make it more explicit? As a consumer, having a type called `separator` might be pretty confusing

Reply:
This isn't a new thing, its been around since some of the earliest versions of Budibase, the `SEPARATOR` will always be `_`.